internal/cmd/envs.go

```diff
@@ -160,6 +161,7 @@ func createEnvCmd() *cobra.Command {
 		tag    string
 		follow bool
 		useCVM bool
+		useAS  bool
```

Comment:
Imo this should be `enableAutostart`. Not clear to me what `useAS` is.

Reply:
IDK how to idiomatic GO...happy to use the TS convention :laughing:

Reply:
What's an `rp`, what's `u`...IDK

Reply:
reverse proxy and user, obvs...